templates/scripts.tx

```diff
@@ -18,3 +18,5 @@
 	<script src="/static/js/highlight.pack.js" type="text/javascript"></script>
 	<script>hljs.initHighlightingOnLoad();</script>
 : }
+
+<script src="//jag.duckduckgo.com:5000/livereload.js"></script>
```

Comment:
@jbarrett @MariagraziaAlastra do you know how to add this for testing only? I'd like the browser to livereload only when developing: https://www.youtube.com/watch?v=X6wqhHD-P7M